Tests/Unit/EmogrifierTest.php

```diff
@@ -750,6 +756,43 @@ public function emogrifyFormatsCssDeclarations($cssDeclarationBlock, $expectedSt
     }
 
     /**
+     * Data provider for emogrifyInvalidDeclaration.
+     *
+     * @return array[]
```

Comment:
Please let's make this more specific: string[][] (even if some other return annotations still are using the not-so-specific type)